src/components/_modal/pages/deleteOrder/DeleteOrder.stories.js

```diff
@@ -43,11 +45,13 @@ export function DeleteOrderReadyToPickUp() {
 
   // dummy context for receipt
   const context = {
-    label: Translate({
-      context: "profile",
-      label: "delete-order",
-    }),
+    label:
+      Translate({
+        context: "profile",
+        label: "delete-order",
+      }) + "?",
```

Comment:
Måske skal "?" ind i translation? Er det ikke bedst practice i virkeligheden, at alt text er styret derfra. 

Lidt tænkt eksempel der nok aldrig bliver virkelighed, men hvis nu det blev spansk "¿por que?"

Reply:
Jeg var også i tvivl her, fordi delete-order oversættelsen bliver også brugt uden spørgsmålstegn, som så betyder at man skal rette teksten to steder fremadrettet. Men ja, ingen store følelser her. Jeg retter det bare til.